test/evergreen.yml

```diff
@@ -2659,6 +2676,18 @@ buildvariants:
     - name: ".stress-test-3"
     - name: ".stress-test-4"
 
+- name: large-scale-tests
```

Comment:
can we call it "performance tests", this will end up our perf tests on evergreen

Reply:
As discussed, will be renamed at a later stage if needed. Good for now.